tests/unit/server/bundle_manager/__init__.py

```diff
@@ -158,8 +158,10 @@ def create_run_bundle(self, state=State.CREATED, metadata=None):
             uuid=generate_uuid(),
             state=state,
         )
-        bundle.is_frozen = None
+        bundle.frozen = None
```

Comment:
We had some errors with test setup that only surfaced when writing tests for this PR.